src/Mvc/test/WebSites/HtmlGenerationWebSite/Startup.cs

```diff
@@ -19,7 +19,6 @@ public void ConfigureServices(IServiceCollection services)
             // null which is interpreted as true unless element includes an action attribute.
             services.AddMvc(ConfigureMvcOptions)
                 .InitializeTagHelper<FormTagHelper>((helper, _) => helper.Antiforgery = false)
-                .AddNewtonsoftJson()
```

Comment:
We were using AddNewtonsoftJson to get an implementation of the tempdata serializer. We no longer need it now.